Assets/Animation/Animation/ClimbUp - Copy.anim

```diff
@@ -0,0 +1,63 @@
+%YAML 1.1
```

Comment:
Ik was anims aan het kopieren zodat ik de timeframes in de controller van fiona kon matchen, heb die even over het hoofd gezien ;(